sonar-docs/analysis/scan/sonarscanner-for-msbuild.md

```diff
@@ -0,0 +1,203 @@
+* Single .NET Core project files (csproj or vbproj) could be built and successfully analyzed only if a `<ProjectGuid>unique guid</ProjectGuid>` element is added in the csproj or vbproj XML. The `<ProjectGuid>` element is not required if you build a solution (sln) containing that project.
+
+## Analysis steps
+### Begin
+The begin step is executed when you add the `begin` command line argument. It hooks into the MSBuild pipeline, downloads SonarQube quality profiles and settings and prepares your project for the analysis. 
+
+Command Line Parameters:
+
+Parameter|Description
+---|---
+`/k:<project-key>`|[required] Specifies the key of the analyzed project in SonarQube
+`/n:<project name>`|[optional] Specifies the name of the analyzed project in SonarQube. Adding this argument will overwrite the project name in SonarQube if it already exists.
+`/v:<version>`|[recommended] Specifies the version of your project.
+`/d:sonar.login=<username> or <token>`| [optional] Specifies the username or access token to authenticate with to SonarQube. If this argument is added to the begin step, it must also be added on the end step.
+`/d:sonar.password=<password>`|[optional] Specifies the password for the SonarQube username in the `sonar.login` argument. This argument is not needed if you use authentication token. If this argument is added to the begin step, it must also be added on the end step.
+`/d:sonar.verbose=true`|[optional] Sets the logging verbosity to detailed. Add this argument before sending logs for troubleshooting.
+`/d:<analysis-parameter>=<value>`|[optional] Specifies an additional SonarQube [analysis parameter](/analysis/analysis-parameters/), you can add this argument multiple times. 
+
+For detailed information about all available parameters, see [Analysis Parameters](/analysis/analysis-parameters/).
+
+[[warning]]
+| ![](/images/exclamation.svg) The "begin" step will modify your build like this:
+| * the active `CodeAnalysisRuleSet` will be updated to match the SonarQube quality profile
+| * `WarningsAsErrors` will be turned off
+|
+| If your build process cannot tolerate these changes we recommend creating a second build job for SonarQube analysis.
+
+### Build
+Between the `begin` and `end` steps, you need to build your project, execute tests and generate code coverage data. This part is specific to your needs and it is not detailed here.
+
+### End
+The end step is executed when you add the "end" command line argument. It cleans the MSBuild hooks, collects the analysis data generated by the build, the test results, the code coverage and then uploads everything to SonarQube.
+
+There are only two additional arguments that are allowed for the end step:
+
+Parameter|Description
+---|---
+`/d:sonar.login=<username> or <token>`|[optional] This argument is required if it is added to the begin step.
+`/d:sonar.password=<password>`|[optional] This argument is required if it is added to the begin step and not required if you are using <token>
```

Comment:
This text is missing completely when viewed in GitHub because of the angle brackets. Does it render correctly in the embedded docs view?

Either way it might be worth enclosing it in backticks so the angle-brackets are rendered.

Reply:
for reference: https://next.sonarqube.com/sonarqube/documentation/analysis/scan/sonarscanner-for-msbuild/

Reply:
Thanks. So the word _token_ isn't rendered correctly in `if you are using <token>` in our generated docs either.

Reply:
reworded altogether